encoding/x/protobuf/types.go

```diff
@@ -33,7 +33,7 @@ import (
 
 const (
 	// Encoding is the name of this encoding.
-	Encoding transport.Encoding = "protobuf"
+	Encoding transport.Encoding = "proto"
```

Comment:
WHOOOOOOOPS! so this is a wire protocol breaking change.  I deployed a dev version of yarpc 1.9 to demo-yarpc-go and it caused bad request errors because it couldn't determine the procedure anymore.